model/distro/distro.go

```diff
@@ -896,6 +896,9 @@ func GetHostCreateDistro(ctx context.Context, createHost apimodels.CreateHost) (
 
 // GetDistrosForImage returns the distros for a given image.
 func GetDistrosForImage(ctx context.Context, imageID string) ([]Distro, error) {
+	if imageID == "" {
+		return nil, errors.Errorf("imageID not found")
```

Comment:
nit: when something isnt provided, it's not the same as not being found. The error should just say something like image id not provided